pulseapi/entries/migrations/0006_auto_20170525_2235.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't quite understand where this migration comes from - I pulled master, then checked out a new branch, this migration just gets invented.